tests/browser/test/test_setup.js

```diff
@@ -367,8 +366,8 @@ async function connect(
   );
 
   const delta = {
-    x: targetLocation.x - draggedLocation.x,
-    y: targetLocation.y - draggedLocation.y,
+    x: Math.round(targetLocation.x - draggedLocation.x),
+    y: Math.round(targetLocation.y - draggedLocation.y),
```

Comment:
Can you add an explanation for this change to the PR description please?  It seems unrelated to the config changes but… is needed to get certain tests to pass in certain circumstances?

Reply:
Done. As I added above:

Add some Math.round calls to deal with non-integer pixel values.
Required because webdriverio got stricter and I missed pulling it into https://github.com/google/blockly/pull/7493